src/containers/ListingPage/ListingPage.module.css

```diff
@@ -95,15 +102,61 @@
   cursor: initial;
 }
 
+.actionBarCTA {
+  composes: buttonDefault from global;
+  display: inline-block;
+  min-height: 32px;
+  font-size: 14px;
```

Comment:
Don't set font-size here if you can avoid it. This should come from buttonDefault. (markertplaceDefaults.css)